packages/react-native-win32/src/Libraries/Network/RCTNetworking.win32.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
The TS we had before was just this but missing a couple changes. I replaced this with the Android version with changes to native modules (didn't move from batched bridge to TurboModule syntax yet though)